chatterbot/storage/django_storage.py

```diff
@@ -1,7 +1,5 @@
-import json
 from chatterbot.storage import StorageAdapter
 
-
```

Comment:
Thanks Master!. I'll made my changes through [PyCharm](https://www.jetbrains.com/pycharm/). I found other issues with Train  ``python manage.py train`` with ``import_module`` option isn't working train command. it was failing always. The workaround I have made like this

I tried figured out bug, i couldn't, could you please let me know why it failin and how to solve this problem?

``` Python
CHATTERBOT_TRAIN = {
    'name': 'ADTRAN Chatter Bot',
    'trainer': 'chatterbot.trainers.ChatterBotCorpusTrainer',
    'training_data': [
        './data/greetings.corpus.json'
    ],
    'storage_adapter': 'chatterbot.storage.DjangoStorageAdapter',
    'django_app_name': 'howdy',
    'use_django_models': True
}
# ChatterBot settings
CHATTERBOT = {
    'name': 'Test Chatter Bot',
    'logic_adapters': [
        {
            'import_path': 'chatterbot.logic.BestMatch',
        },
        {
            'import_path': 'chatterbot.logic.LowConfidenceAdapter',
            'threshold': 0.75,
            'default_response': 'I am sorry, but I do not understand.'
        },
        {
            'import_path': 'chatterbot.logic.SpecificResponseAdapter',
            'input_text': 'Help me!',
            'output_text': 'Ok, here is a link: http://chatterbot.readthedocs.io/en/stable/'
        }
    ],
    'filters': [
            'chatterbot.filters.RepetitiveResponseFilter'
    ],
    'storage_adapter': 'chatterbot.storage.DjangoStorageAdapter',
    'django_app_name': 'howdy',
    'use_django_models': True
}
```